include/oneapi/dpl/pstl/onedpl_config.h

```diff
@@ -44,7 +44,7 @@
 
 #if defined(ONEDPL_USE_PREDEFINED_POLICIES)
 #    undef _ONEDPL_USE_PREDEFINED_POLICIES
-#    define _ONEDPL_USE_PREDEFINED_POLICIES ONEDPL_USE_DEFAULT_POLICIES
+#    define _ONEDPL_USE_PREDEFINED_POLICIES ONEDPL_USE_PREDEFINED_POLICIES
```

Comment:
`_ONEDPL_USE_PREDEFINED_POLICIES` this is the only macro that remains with `_ONEDPL` prefix with `USE` in the middle. I would like to rename it to something more "safe" from mistake perspective. I am not saying you should this @akukanov

Reply:
I see no problem in the ``_ONEDPL_USE`` prefix combination. Actually, I'd prefer that in the following line (that can be found below in the file) there was this suffix as well, instead of just ``_USE``:
```
# define _USE_RADIX_SORT (_USE_SUB_GROUPS && _USE_GROUP_ALGOS)
```